resources/views/admin/events/participants.blade.php

```diff
@@ -33,11 +33,8 @@
         </div>
         <div class="row">
             <div class="col-12 mt-3">
-            @if ($participants)
-                @if ($participants->isEmpty())
-                    <p>No participants.</p>
-                @else
-                <table class="user-table text-center table-bordered">
+                @if ($participants && $participants->isNotEmpty())
```

Comment:
you can also make $participants->isnotempty only since that is the one will check if we need to place a data

Reply:
I'll change it.